e-store/e-store/ScenesScreen/Home/HomeViewController/HomeViewController.swift

```diff
@@ -11,11 +11,33 @@ class HomeViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        setupNavigationItems()
         // Do any additional setup after loading the view.
     }
 
-
+    private func setupNavigationItems() {
+        navigationItem.title = "Mega Mall"
+        let bell = UIBarButtonItem(image: UIImage(systemName: "bell"),
```

Comment:
plz use design images because system images support only iOS 13 and later

Reply:
what is design image?